layout.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
```

Comment:
why was this changed?

Reply:
![image](https://github.com/user-attachments/assets/22deeae8-b6e1-4759-98ff-bd28a842ae99)

When we were trying to push this commit, we kept having a prettier format error for this file. This is what it recommended that we do, to update to a lowercase `doctype`. From my understanding, there is no functional difference between the two.